tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -250,7 +250,6 @@ generate_qemu_options() {
 	qemu_options+=(size:--disable-auth-pam)
 
 	# Disable unused filesystem support
-	[ "$arch" == x86_64 ] && qemu_options+=(size:--disable-fdt)
```

Comment:
Is this dropped because fdt/tcmalloc are deprecated in 6.2? is it worth hiding this behind a version check?

Reply:
> Is this dropped because fdt/tcmalloc are deprecated in 6.2?

@egernst It is dropped because the x86_64-softmmu target requires libfdt since [commit](https://gitlab.com/qemu-project/qemu/-/commit/f5918a992839454c6e87234760bdf7aee804aa33):
```
commit f5918a992839454c6e87234760bdf7aee804aa33
Author: Gerd Hoffmann <kraxel@redhat.com>
Date:   Thu Oct 14 21:36:17 2021 +0200

    microvm: add device tree support.
    
    Allows edk2 detect virtio-mmio devices and pcie ecam.
    See comment in hw/i386/microvm-dt.c for more details.
    
    Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
    Reviewed-by: Sergio Lopez <slp@redhat.com>
    Message-Id: <20211014193617.2475578-1-kraxel@redhat.com>
```

BTW fdt isn't a filesystem. It thus looks a bit weird that the line was placed in the `# Disable unused filesystem support` section.

> is it worth hiding this behind a version check?

I don't think so because x86_64-softmmu didn't require libfdt with older versions of QEMU, i.e. `--disable-fdt` was the default behaviour.